lib/streamlit/elements/image.py

```diff
@@ -53,7 +53,8 @@
 ImageOrImageList: TypeAlias = Union[AtomicImage, List[AtomicImage]]
 UseColumnWith: TypeAlias = Optional[Union[Literal["auto", "always", "never"], bool]]
 Channels: TypeAlias = Literal["RGB", "BGR"]
-OutputFormat: TypeAlias = Literal["JPEG", "PNG", "auto"]
+ImageFormatOrAuto: TypeAlias = Literal["JPEG", "PNG", "auto"]
+ImageFormat: TypeAlias = Literal["JPEG", "PNG"]
```

Comment:
Nit: You can define `ImageFormatOrAuto` in terms of `ImageFormat`:

```py
ImageFormat: TypeAlias = Literal["JPEG", "PNG"]
ImageFormatOrAuto: TypeAlias = Literal[ImageFormat, "auto"]
```

One could argue that it is slightly less readable, but the benefit is that the types are more robust and easier to maintain, as you can't change one and not the other.

Reply:
I didn't realize this - and yeah I agree!